index.html

```diff
@@ -11,47 +11,299 @@
+                                <p class="text">JavaScript</p>
+                            </li>
+                            <li class="content">
+                                <img class="icon" src="images/HTML-logo.png" alt="html-logo">
+                                <p class="text">HTML</p>
+                            </li>
+                            <li class="content">
+                                <img class="icon" src="Images/CSS-logo.png" alt="css-logo">
+                                <p class="text">CSS</p>
+                            </li>
+                        </ul>
+                    </li>
+                    <li>
+                        <div class="frame-7">
+                            <h4 class="languages">Frameworks</h4>
+                            <a class="arrow-down" href="#"><img src="Images/arrow-right.png" alt="Arrow Right Icon">
+                            </a>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="frame-7">
+                            <h4 class="languages">Skills</h4>
+                            <a class="arrow-down" href="#"><img src="Images/arrow-right.png" alt="Arrow Right Icon"></a>
+                        </div>
+                    </li>
+                </ul>
+            </div>
         </section> 
-         
-    
+        <!-- contact form -->
+        <section class="contact-me">
+            
+                <h2 class="heading-2">Contact me</h2>
+                <p class="wordings">
+                  If you have an application you are interested in developing, a feature
+                  that you need built or a project that needs coding. I’d love to help
+                  with it
+                </p>
+                <form
+                  action="https://formspree.io/f/mgedjkyb"
+                  id="form1"
+                  method="post"
+                  class="contact-form"
+                >
+                  <div class="input-control">
+                    <input
+                      type="text"
+                      id="name"
+                      placeholder="Your Name"
+                      name="name"
+                      maxlength="30"
+                      required
+                    />
+                  </div>
+                  <div class="input-control">
+                    <input
+                      type="email"
+                      id="email"
+                      placeholder="Your Email"
+                      name="email"
+                      required
+                    />
+                  </div>
+                  <div class="input-control">
+                    <textarea
+                      placeholder="Write your message here"
+                      id="message"
+                      name="message"
+                      maxlength="500"
+                      required
+                    ></textarea>
```

Comment:
- [ OPTIONAL ] I think it would be better if you add the correct `font-family` on `textearea` field  text  like the one used on the Figma design